src/elements/button/button.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 angular
-  .module('semantic.ui.elements.button', [])
+  .module('semantic.ui.elements.button', ['semantic.ui.elements.icon', 'semantic.ui.elements.divider'])
```

Comment:
Sorry, I'm a bit new to PRs, when I tried creating a new PR, for the buttons module, it got merged with the current one :(